src/website/views/components/forms/checkbox/_properties.pug

```diff
@@ -1,6 +1,29 @@
 h2 Web Component
 h3 Properties
-.chi-card.-align--center.-empty
```

Comment:
Please, revert the change as JS property indeterminate is generic, not Chi specific.

Reply:
Done.